dkms.in

```diff
@@ -1670,7 +1670,7 @@ do_status_weak()
 # interested in, but that the DKMS internals do not usually care about.
 module_status_built_extra() (
     set_module_suffix "$3"
-    read_conf "$3" "$4" "$dkms_tree/$1/$2/source/dkms.conf"
+    read_conf "$3" "$4" "$dkms_tree/$1/$2/source/dkms.conf" 2>/dev/null
```

Comment:
Why are we masking only this one and not the other dozen+ instances of read_conf/read_conf_or_die?